src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -58150,5 +58150,256 @@ public interface ITest33
             }
         }
 
+        [Fact]
+        public void LambdaCaching_01()
+        {
+            var source0 = @"
+interface I1
+{
+    static void M1()
+    {
+        M2(() => System.Console.WriteLine(""M1""));
+    }
+
+    static void M2(System.Action d)
+    {
+        d();
+    }
+}
+
+class A
+{
+    static void Main()
+    {
+        I1.M1();
+    }
+}
+";
+
+            var compilation0 = CreateCompilation(source0, options: TestOptions.DebugExe, targetFramework: TargetFramework.NetStandardLatest);
+            compilation0.VerifyDiagnostics();
+
+            var verifier = CompileAndVerify(compilation0, expectedOutput: !ExecutionConditionUtil.IsMonoOrCoreClr ? null : @"M1", verify: VerifyOnMonoOrCoreClr);
```

Comment:
>!ExecutionConditionUtil.IsMonoOrCoreClr ? null : @"M1" [](start = 74, length = 54)

Consider inverting this conditional to make it simpler to read.

Reply:
> Consider inverting this conditional to make it simpler to read.

I find it  much easier to have the string at the end, especially when it is multi-line.